### PR TITLE
Multiprocessing Bug

### PR DESCRIPTION
**Pull request by samihamdan**
We had a small bug with using multiprocessing and using the `wrap_search` function. 
Because `wrap_search` is not pickable, it was not being usable by multiprocessing in sklearns parallel_backend.

### CODE REVIEW

File: docs/changes/latest.inc [hunk 1]
```diff
@@ -32,5 +32,7 @@ Bugs
 
 - Fix compatibility with new scikit-learn release 0.24 (:gh:`#108` by `Sami Hamdan`_)
 
+- Fix compatibility with multiprocessing in sklearn (by `Sami Hamdan`_)
```
Comment (fraimondo) [inline docs/changes/latest.inc:35]: let's use scikit-learn and not sklearn.

File: julearn/scoring.py [hunk 2]
```diff
@@ -26,6 +26,9 @@ def get_extended_scorer(estimator, score_name):
     scorer = check_scoring(estimator, scoring=score_name)
 
     def extended_scorer(estimator, X, y):
-        y_true = estimator.transform_target(X, y)
+        try:
+            y_true = estimator.transform_target(X, y)
+        except AttributeError:
```
Comment (fraimondo) [inline julearn/scoring.py:31]: I personally do not agree to use exceptions to handle code flow. The correct way here will be to use `hasattr(estimator, 'transform_target')`

**Review by fraimondo — CHANGES_REQUESTED** [review]

**Comment by fraimondo** [comment]
Just asked for two minor changes. But despite that, good work!

**Comment by samihamdan** [comment]
thx, changes should be implemented now.

**Review by fraimondo — APPROVED** [review]

**Pull request merged**